bentoml/_internal/cli/click_utils.py

```diff
@@ -88,15 +90,14 @@ def wrapper(
                 load_global_config(config)
 
             if quiet:
-                logging.getLogger("bentoml").setLevel(logging.ERROR)
+                set_quiet_mode(True)
                 if verbose:
-                    logger.warning(
-                        "The bentoml command option `--verbose/--debug` is ignored when"
-                        "the `--quiet` flag is also in use"
-                    )
+                    logger.warning("'--quiet' passed; ignoring '--verbose/--debug'")
             elif verbose:
                 set_debug_mode(True)
 
+            configure_logging()
```

Comment:
does this mean that when user use our API from python, logging won't get configured?

Reply:
Indeed.

Reply:
should we also configure logs for python API separately?

Reply:
I believe the decision was made because logging configuration is mainly an application's job and not a library's; https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library.